misc/python/materialize/mzexplore/extract.py

```diff
@@ -160,11 +173,14 @@ def plans(
             require_ssl=db_require_ssl,
         )
     ) as db:
-        for item in db.catalog_items(database, schema, name):
+        for item in db.catalog_items(database, schema, name, system):
             item_database = sql.identifier(item["database"])
             item_schema = sql.identifier(item["schema"])
             item_name = sql.identifier(item["name"])
-            fqname = f"{item_database}.{item_schema}.{item_name}"
+            if item["database"] == "mz":  # don't prepend pseudo-database `mz`
```

Comment:
Just for my own edification: when does this happen?

Reply:
This is because the system tables in the catalog belong to the ephemeral database (that is, their FQ names are `schema.item` and they should be available from every database). This is modeled somewhat awkwardly in the `s_items.sql` query where I select `'mz' as database` for consistency with the `u_items.sql` result.